yaml/places.yaml

```diff
@@ -34,6 +34,38 @@ global:
         func: util.true_or_none
         args: [ { col: is_landuse_aoi } ]
     wikidata_id: {col: wikidata}
+    disputed_by: {col: disputed_by }
+    'place:ISO': { col: 'place:ISO' }
```

Comment:
Consider cleaning up so it's more like Natural Earth POVs.

Reply:
Specifically:

A list of country codes:
- https://github.com/tilezen/vector-datasource/blob/master/queries/ne.jinja2#L2

And then function to expand them:
- https://github.com/tilezen/vector-datasource/blob/master/queries/ne.jinja2#L28-L32

Might require a slight refactor, but be cleaner?

Reply:
I'm aware of the jinja, and it's definitely my strong preference that it be done this way if possible.  But, based on the existence of the fclass thing for places and boundaries (e.g. https://github.com/tilezen/vector-datasource/blob/5afa684b54de99fde44d4c489140458091173b7e/yaml/places.yaml#L85-L115), I assumed that I would still need an entry like this.  Do you think otherwise?

Reply:
If by 'slight refactor' you mean to add support for this in our yaml DSL in kinds.py, that's a can of worms and I'd prefer to not gate any of this on that change

Reply:
Sure